bears/documentation/DocumentationStyleBear.py

```diff
@@ -24,9 +24,11 @@ def process_documentation(self,
                               doc_comment: DocumentationComment,
                               allow_missing_func_desc: str=False,
                               indent_size: int=4,
-                              expand_one_liners: str=False):
+                              expand_one_liners: str=False,
+                              use_spaces: bool=True):
         """
-        This fixes the parsed documentation comment.
+        This fixes the parsed documentation comment. In case you want to
+        have a indentation of 2 tabs, it is not possible.
```

Comment:
"In case you want to have a indentation of 2 tabs, it is not possible" doesn't apply any more, you allow this.